modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -0,0 +1,81 @@
+[id="cluster-logging-release-notes-5-1-0"]
+= OpenShift Logging 5.1.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-2021:2112[RHSA-2021:2112 OpenShift Logging Bug Fix Release 5.1.0].
+
+[id="openshift-logging-5-1-0-new-features-and-enhancements"]
+== New features and enhancements
+
+OpenShift Logging 5.1 now supports {product-title} 4.7 and later running on:
```

Comment:
@rolfedh thats the statement i mentioned on slack.

Reply:
@sichvoge The sentence says "4.7 and later", which remains true as of today.

Are you saying that if we do not go GA by tomorrow, then we may have to replace 4.7 with 4.8? Please confirm. Thanks!